src/backend/dx11/src/lib.rs

```diff
@@ -358,6 +360,178 @@ impl Device {
             context
         }
     }
+
+    fn create_rasterizer_state(&self, rasterizer_desc: &pso::Rasterizer) -> Result<ComPtr<d3d11::ID3D11RasterizerState>, pso::CreationError> {
```

Comment:
let's stop extending the device in "lib.rs" and do it in "device.rs" like all other backends do

Reply:
I should probably get rid of the files which are leftover from before too.

Do you mind if I do this on the next PR? I have some commits which are based off of this branch already and can see the merge conflicts happening already in my mind..

Reply:
sure